src/driver/ydb/YdbQueryRunner.ts

```diff
@@ -12,52 +12,82 @@ import { View } from "../../schema-builder/view/View"
 import { IsolationLevel } from "../types/IsolationLevel"
 import { YdbDriver } from "./YdbDriver"
 import { ReplicationMode } from "../types/ReplicationMode"
+import { QueryRunnerAlreadyReleasedError } from "../../error"
+import { Broadcaster } from "../../subscriber/Broadcaster"
 
 export class YdbQueryRunner extends BaseQueryRunner implements QueryRunner {
+    /**
+     * Database driver used by connection.
+     */
+    driver: YdbDriver
+
     constructor(ydbDriver: YdbDriver, replicationMode: ReplicationMode) {
         super()
+        this.driver = ydbDriver
+        this.connection = ydbDriver.connection
+        this.broadcaster = new Broadcaster(this)
```

Comment:
What is it for?

Reply:
It is for event broadcasting
https://typeorm.io/listeners-and-subscribers